Objects/genericaliasobject.c

```diff
@@ -234,6 +263,7 @@ ga_getitem(PyObject *self, PyObject *item)
     }
     int is_tuple = PyTuple_Check(item);
     Py_ssize_t nitem = is_tuple ? PyTuple_GET_SIZE(item) : 1;
+    PyObject **args = is_tuple ? &PyTuple_GET_ITEM(item, 0) : &item;
```

Comment:
Maybe rename this to `items`? Or `argitems`? I was quite confused for a while by the distinction between `alias->args` (the `__args__` attribute) vs. `args` (the arguments to `__getitem__`).

Reply:
Done.